src/material/schematics/ng-generate/mdc-migration/rules/theming-styles.ts

```diff
@@ -14,15 +14,35 @@ import {ComponentMigrator, MIGRATORS} from '.';
 
 const COMPONENTS_MIXIN_NAME = /\.([^(;]*)/;
 
+/**
+ * Mapping between the renamed legacy typography levels and their new non-legacy names. Based on
+ * the mappings in `private-typography-to-2018-config` from `core/typography/_typography.scss`.
+ */
+const RENAMED_TYPOGRAPHY_LEVELS = new Map([
```

Comment:
I know the non MDC typography levels have `input` but is not in the new one/being mapped. Is it just not able to be mapped? If so, we should probably add it somewhere in the migration doc.

Reply:
Ah, good catch. I'm leaning more towards adding an `input` level to the MDC config. I'll merge in the PR as is for now and will ping Miles when he's back since he set up the MDC configs.

Reply:
Resolution: the schematic should remap `input` to `body-1`.